contents/docs/getting-started/person-properties.mdx

```diff
@@ -15,7 +15,9 @@ import UserPropertiesSetVsSetOnce from "./\_snippets/user-properties-set-vs-set-
 
 For example, you can create [filters](/docs/product-analytics/trends#filtering-events-based-on-properties) or [cohorts](/docs/data/cohorts) based on person properties, which can then be used to [create insights](/docs/product-analytics/insights), [feature flags](/docs/feature-flags), and more. 
 
-For backward compatibility, PostHog captures [identified events](/docs/data/anonymous-vs-identified-events) that create a [person profile](/docs/data/persons) by default. You can change this by setting your snippet or frontend SDK initialization's `person_profiles` value to `identified_only` or a server-side SDK or API event's `$process_person_profile` property to `false`.
+By default, PostHog's client-side SDKs will send [anonymous events](/docs/data/anonymous-vs-identified-events), meaning that your users will not have a [person profile](/docs/data/persons), unless you call `.identify()`, `.set()`, or another method that uses person profiles. You can change this by setting your snippet or frontend SDK initialization's `person_profiles` value to `always`
```

Comment:
I've been dragging my feet on updating all of these because the default **hasn't changed**. We are inconsistent here.

Source: https://posthog.slack.com/archives/C06R39WNY5B/p1732255387985419

Reply:
Oh my bad, I was basing this off https://posthog.slack.com/archives/C02E3BKC78F/p1732646741093759?thread_ts=1732640114.562429&cid=C02E3BKC78F

Reply:
Hmm, that one is newer so we can merge this then.

Reply:
Tbh I'd rather leave it at this point, we can get clarity on monday when people are back from thanksgiving